01-messaging.md

```diff
@@ -9,7 +9,7 @@ The default TCP port depends on the network used. The most common networks are:
 
 - Bitcoin mainnet with port number 9735 or the corresponding hexadecimal `0x2607`;
 - Bitcoin testnet with port number 19735 (`0x4D17`);
-- Bitcoin signet with port number 39735 (`0xF87`).
+- Bitcoin signet with port number 39735 (`0x9B37`).
```

Comment:
Ok this was my calculation, so if it is wrong is my fault, but `(F87)₁₆ = (15 × 16²) + (8 × 16¹) + (7 × 16⁰) = (3975)₁₀` is `3975 = 3840 + 128 + 7`

Looks correct to me

Reply:
@vincenzopalazzo  It says the port it `39735`, not `3975`

Reply:
Ops! :) @JssDWt thanks!